packages/terra-graphs-docs/src/example-datasets/popup.js

```diff
@@ -0,0 +1,291 @@
+/* eslint-disable */
+import * as d3 from "d3";
+import constants from "@cerner/carbon-graphs/src/js/helpers/constants";
+import utils from "@cerner/carbon-graphs/src/js/helpers/utils";
+
+const getDate = (d) => new Date(d).toLocaleString();
+const getPairData = (value) =>
+    value ? (checkDate(value) ? getDate(value) : value) : null;
+const checkDate = (date) => date instanceof Date;
+const createItem = (pair, label, value) => {
+    const item = pair.append("g").classed("popup-item", true);
+    item.append("label").classed("popup-label", true).text(label);
+    item.append("g").classed("popup-text", true).text(value);
+};
+const createTrackItem = (pair, label, items) => {
```

Comment:
I made a similar comment on a separate graph pull request, but I think converting most of these functions into an object based api will really help clarify what each function argument represents.
```suggestion
const createTrackItem = ({ pair, label, items }) => {

// useage
createTrackItem({ pair: element, label: 'real-world-label', items: dataArr });
```

Reply:
I agree, but I think that's beyond the scope of this issue. The library has some questionable design choices and refactoring those is a separate project that will take more than an iteration. I'd rather avoid making api design changes until we're ready to tackle it as dedicated project due to the spaghetti nature of the implementation and avoid breaking the library.

Reply:
Here are some key questions - is this pull request moving old code into the terra-graphs repo, or are these functions brand new on your side? If it's the former then I agree with you, but if it's the latter, I disagree and think it's in our best interest to update the api. 

Writing some context in the pr description would help everyone understand the intent of the pull request. Like "migrating bubble examples from carbon into terra-graphs to render on dev site for integration tests"

Reply:
I apologize for the lack of clarity. These functions and datasets are being migrated from the old dev-site.